packages/react/src/components/table-hoc/TableWithFilter.tsx

```diff
@@ -37,13 +38,15 @@ export const tableWithFilter = (
     const mapStateToProps = (state: PlasmaState, ownProps: OwnProps) => {
         const filterText = FilterBoxSelectors.getFilterText(state, ownProps);
         const matchFilter = ownProps.filterMatcher || config.matchFilter || defaultMatchFilter;
+        const isFocus = ownProps.isFilterInFocus ? true : false;

```

Comment:
To replace the ternary which I don't find necessary on a boolean finally if it can be undefined at this place there

```suggestion
        const isFocus = !!ownProps.isFilterInFocus;
```

Reply:
already fixed :speak_no_evil:

Reply:
Tu vas trop vite pour moi :D